packages/task/backend/sirius-components-task-edit/src/main/java/org/eclipse/sirius/components/task/configuration/TaskEditEMFConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.task.configuration;
+
+import org.eclipse.emf.common.notify.AdapterFactory;
+import org.eclipse.sirius.components.task.provider.TaskItemProviderAdapterFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration of the EMF support for Task MM.
+ *
+ * @author lfasani
+ */
+@Configuration
+public class TaskEditEMFConfiguration {
+    @Bean
+    AdapterFactory taskAdapterFactory() {
+        return new TaskItemProviderAdapterFactory();
+    }
+}
```

Comment:
- There should not be any dependency to spring in the code of the metamodels
- Why isn't there any visibility on this method, is checkstyle not detecting that?

Reply:
For those who may be wondering, Checkstyle is not running here because we take it for granted that no one will add anything in the metamodels

Reply:
A ticket is alredy opened to handle that